llmproxy/llmproxy.py

```
@@ -163,12 +163,10 @@ def _cost_route(self, prompt: str):
             # Attempt to make request to model
             try:
```

This code needs to be refactored to throw error instead of returning